src/Cardano/Types/Value.purs

```diff
@@ -220,7 +221,7 @@ unsafeAdaSymbol = CurrencySymbol mempty
 -- | constructor is not exported
 mkCurrencySymbol :: ByteArray -> Maybe CurrencySymbol
 mkCurrencySymbol byteArr =
-  scriptHashFromBytes byteArr *> pure (CurrencySymbol byteArr)
+  scriptHashFromBytes (wrap byteArr) *> pure (CurrencySymbol byteArr)
```

Comment:
I know this wasn't you, but I think:

```suggestion
  scriptHashFromBytes (wrap byteArr) $> CurrencySymbol byteArr
```
is nicer

Reply:
Actually I'm going to open a PR with some other stylistic changes for things that were recently discovered, I can take care of this then